mathquill4quill.js

```diff
@@ -240,14 +268,121 @@ window.mathquill4quill = function(dependencies) {
       };
     }
 
+    function newHistoryList() {
+      const history = historyList || [];
+      let historyDiv = null;
+
+      function applyHistoryButtonStyles(button) {
+        button.style.overflow = "hidden";
+        button.style.margin = "5px";
+        button.style.width = "270px";
+        button.style.height = "65px";
+        button.style.minHeight = "60px";
+        button.style.backgroundColor = "#ffffff";
+        button.style.borderColor = "#000000";
+        button.style.borderRadius = "7px";
+        button.style.borderWidth = "2px";
+        button.style.cursor = "pointer";
+        button.style.transition = "background-color 0.3s linear";
```

Comment:
The styles were recently moved in pull request #36 from within the mathquill4quill.js file to a seperate style sheet. It would be great if you could move these styles over to this style sheet.